libcontainer/cgroups/fs2/rdma.go

```diff
@@ -0,0 +1,17 @@
+// +build linux
+
+package fs2
+
+import (
+	"github.com/opencontainers/runc/libcontainer/cgroups"
+	"github.com/opencontainers/runc/libcontainer/cgroups/fscommon"
+	"github.com/opencontainers/runc/libcontainer/configs"
+)
+
+func setRdma(path string, r *configs.Resources) error {
+	return fscommon.RdmaSet(path, r)
+}
+
+func statRdma(path string, stats *cgroups.Stats) error {
+	return fscommon.RdmaGetStats(path, stats)
+}
```

Comment:
It seems this whole file is useless -- you can just call `fscommon.Rdma*` functions directly in `fs2/fs2.go`

Reply:
@kolyshkin Resolved thanks for spotting this.